docs/snippets/common/checkbox-story.mdx.mdx

```diff
@@ -1,6 +1,8 @@
 ```md
 <!--- Checkbox.stories.mdx -->
 
+<!--- This is your Story template function, shown here in React -->
```

Comment:
Can we put this above the `export const Template = ` line?

Reply:
Updated with the latest commit